dask/tests/test_distributed.py

```diff
@@ -25,6 +26,21 @@
     cluster = partial(cluster, should_check_state=False)
 
 
+# TODO: the fixture teardown for `cluster_fixture` is failing periodically with
+# a PermissionError on windows only (in CI). Since this fixture lives in the
+# distributed codebase and is nested within other fixtures we use, it's hard to
+# patch it from the dask codebase. And since the error is during fixture
+# teardown, an xfail won't cut it. As a hack, for now we skip all these tests
+# on windows.
+pytestmark = pytest.mark.skipif(
+    sys.platform == "win32",
```

Comment:
Optional style nit: I wouldn't mind storing what platform is used in variables in `dask/compatibility.py` similar to what `distributed` does [here](https://github.com/dask/distributed/blob/cd2ce86979de76c77ae361e1e03d1f74d3efbf53/distributed/compatibility.py#L10-L12). I often find myself having to look up how to specify "what OS am I on?"